src/main/java/com/nguyenquyhy/discordbridge/utils/ColorUtil.java

```diff
@@ -65,4 +68,11 @@ public static TextColor getColor(Color color) {
         }
         return result;
     }
+    public static String removeColor(String text){
```

Comment:
We would need to rewrite quite a few of the uses of StringUtil in the chat listener, as the majority of that is running with Strings.